PokeApiNet/Models/Pokemon.cs

```diff
@@ -1111,6 +1122,25 @@ public class PokemonSpecies : NamedApiResource
         public List<PokemonSpeciesVariety> Varieties { get; set; }
     }
 
+    public class PokemonSpeciesFlavorTexts
```

Comment:
This looks good, but the name of the class shouldn't be specific to pokemon species, as this new class should be used by [abilities](https://github.com/PokeAPI/pokeapi/blob/a5205b512640bd33684acbbb3f23a8d98aee0cbc/pokemon_v2/serializers.py#L1045), [items](https://github.com/PokeAPI/pokeapi/blob/a5205b512640bd33684acbbb3f23a8d98aee0cbc/pokemon_v2/serializers.py#L1297), [moves](https://github.com/PokeAPI/pokeapi/blob/a5205b512640bd33684acbbb3f23a8d98aee0cbc/pokemon_v2/serializers.py#L2015), and [pokemon species](https://github.com/PokeAPI/pokeapi/blob/a5205b512640bd33684acbbb3f23a8d98aee0cbc/pokemon_v2/serializers.py#L2688). Not sure what to call it though, as we still need the other FlavorText class that excludes the `version` field. What are your thoughts?

Additionally, since this class isn't specific to the pokemon species resource, I'd like to see this new class in Common.cs instead.

Reply:
The linked classes have a `version_group` of type `VersionGroup` instead of a `version` field of type `Version`, or I may seeing that wrong?

Reply:
Whoops, you're right; I completely missed that. In that case, this class is perfect!